modules/elasticsearch/src/test/java/org/testcontainers/elasticsearch/ElasticsearchContainerTest.java

```diff
@@ -92,11 +104,40 @@ public void elasticsearchOssImage() throws IOException {
         }
     }
 
+    @Test
+    public void restClientClusterHealth() throws IOException {
+        // httpClientContainer {
+        // Create the elasticsearch container.
+        try (ElasticsearchContainer container = new ElasticsearchContainer()) {
+            // Start the container. This step might take some time...
+            container.start();
+
+            // Do whatever you want with the rest client ...
+            final CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
+            credentialsProvider.setCredentials(AuthScope.ANY,
+                new UsernamePasswordCredentials(ELASTICSEARCH_USERNAME, ELASTICSEARCH_PASSWORD));
+
+            client = RestClient.builder(HttpHost.create(container.getHttpHostAddress()))
+                .setHttpClientConfigCallback(httpClientBuilder -> httpClientBuilder.setDefaultCredentialsProvider(credentialsProvider))
+                .build();
+
+            Response response = client.performRequest(new Request("GET", "/_cluster/health"));
+            assertThat(response.getStatusLine().getStatusCode(), is(200));
+            assertThat(EntityUtils.toString(response.getEntity()), containsString("cluster_name"));
```

Comment:
We should probably omit these assertions from the docs...

Reply:
is it possible to exclude lines from a code block with the codeinclude macro?
This would be really helpful feature to decrease the amount of new code in the tests

Reply:
Not directly to exclude, but you can use the same block ID more than once and every block should be included. e.g. given:

```
// someBlock {
A
B
// }
C
// someBlock {
D
// }
```

When rendered via codeinclude, it should appear like:
```
A
B
D
```

I'm partly tempted to add explicit excludes support to the codeinclude macro, but I think I'd prefer to try the above approach first.

Reply:
thanks that is what i wanted

Reply:
This PR could help to solve the problem: https://github.com/rnorth/mkdocs-codeinclude-plugin/pull/14

Reply:
I've been trying to untangle the PRs on the codeinclude plugin, because I think you're right. Unfortunately they're not independent, and that PR is built upon other PRs that have issues.

I think it'll be a bit of time before that gets merged.

Shall we just go ahead and merge? The indentation issue should get sorted out separately later 😄

Reply:
sounds like a plan :)